src/gz/gz_macro.c

```diff
@@ -218,6 +218,14 @@ static void save_state_proc(struct menu_item *item, void *data)
   command_savestate();
 }
 
+static void clear_state_proc(struct menu_item *item, void *data)
+{
+    if(gz.state_buf[gz.state_slot]){
```

Comment:
This project uses 2 spaces of indentation. Please change this to be consistent with the rest of the code.
```suggestion
  if (gz.state_buf[gz.state_slot]) {
```